logziosender.go

```diff
@@ -203,25 +257,21 @@ func (l *LogzioSender) start() {
 func (l *LogzioSender) Stop() {
 	defer l.queue.Close()
 	l.Drain()
-
 }
 
 func (l *LogzioSender) tryToSendLogs() int {
 	resp, err := l.httpClient.Post(l.url, "text/plain", l.buf)
```

Comment:
please send data compressed in Gzip 
also add the following headers
https://logzio.atlassian.net/browse/DEV-25393